CMakeLists.txt

```diff
@@ -292,6 +292,10 @@ if (NOT USE_SIMD STREQUAL "")
             else ()
                 add_definitions ("-m${feature}")
             endif ()
+            if (feature STREQUAL "fma" AND CMAKE_COMPILER_IS_CLANG)
+                # Clang needs an explicit flag to make fma ops
+                add_definitions ("-ffp-contract=fast")
```

Comment:
What else does this let the compiler do? This can break other code that relies on IEEE semantics I think.